src/main/resources/assets/securitycraft/models/block/horizontal_reinforced_iron_bars.json

```diff
@@ -0,0 +1,94 @@
+{
+    "ambientocclusion": false,
+    "textures": {
+        "particle": "securitycraft:block/reinforced_iron_bars",
+        "bars": "securitycraft:block/reinforced_iron_bars",
+        "edge": "securitycraft:block/reinforced_iron_bars"
```

Comment:
Is there really a need to have two definitions for the same texture?

Reply:
Not really, just following the reinforced_iron_bars JSON file. Although it's nice to have separate tags to distinguish bars from edges